sympy/integrals/tests/test_integrals.py

```diff
@@ -1026,12 +1022,12 @@ def test_issue_4517():
 
 def test_issue_4527():
     k, m = symbols('k m', integer=True)
-    ans = integrate(sin(k*x)*sin(m*x), (x, 0, pi)
-            ).simplify() == Piecewise(
-        (0, Eq(k, 0) | Eq(m, 0)),
-        (-pi/2, Eq(k, -m)),
-        (pi/2, Eq(k, m)),
-        (0, True))
+    #ans = integrate(sin(k*x)*sin(m*x), (x, 0, pi)
+    #        ).simplify() == Piecewise(
+    #    (0, Eq(k, 0) | Eq(m, 0)),
+    #    (-pi/2, Eq(k, -m)),
+    #    (pi/2, Eq(k, m)),
+    #    (0, True))
```

Comment:
Why is this commented out?

Reply:
It didn't do anything (not an assert). Also, as stated, after simplification it gives (almost) that result. So not obvious was the purpose of the test was or if I should change it to be a test with the "new" result. Or move it to failing tests.